numpyramid.js

```diff
@@ -0,0 +1,25 @@
+var row = require("readline");
+var prompts = row.createInterface(process.stdin, process.stdout);
+prompts.question("How many row you want to print ?", function(n) {
+  for (var i = 0; i <= n; i++) {
+    var sp = " ";
+    for (var space = 0; space <= n - i; space++) {
+      sp = sp + " " + " ";
+    }
+    j = i;
+
+    for (var c = 1; c <= i; c++) {
+      sp = sp + (j + " ");
+      j++;
+    }
+
+    j = j - 2;
+    for (var c = 1; c < i; c++) {
+      sp = sp + (j + " ");
+
+      j--;
+    }
+
+    console.log(sp);
+  }
+});
```

Comment:
This is not working as expected.  If the value of n is 10, the output is not a triangle

Reply:
this happened due to double digits appearing in the pyramid.
resolved the problem, by having only single digit number